sigal/plugins/compress_assets.py

```diff
@@ -0,0 +1,156 @@
+"""Plugin to compress static files for faster HTTP transfer.
+
+Currently, 3 methods are supported:
+    - gzip: No dependency required.
+            This is the fastest, but also largest output.
+    - zopfli: Need zopfli module from https://pypi.python.org/pypi/zopfli.
+              gzip compatible output with optimized size.
+    - brotli: Need brotli module from https://pypi.python.org/pypi/Brotli.
+              Brotli is the best compressor for web usage.
```

Comment:
A small precision for this: if you add your plugin in `docs/plugin.rst` with autodoc (you can look other plugins in this file), then this docstring is used in the HTML doc. So it needs to be valid and well-formatted RST ;)
I can cleanup later if you prefer.